ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -522,9 +523,14 @@ export default class ConfirmTransactionBase extends Component {
         </div>
       ) : null;
 
-    const { simulationData } = txData;
+    const { simulationData, approvalTxId } = txData;
 
-    const simulationDetails = (
+    // if transaction reverts in simulation and there's an approval transaction, we don't want to show the simulation details
+    const isUnsimulable =
+      approvalTxId &&
+      simulationData?.error.code === SimulationErrorCode.Reverted;
```

Comment:
Does the tx get re-simulated after the approval tx is confirmed?

Reply:
Not that I'm aware of, but we set `isUnsimulable` to `true` if the simulation errors with a revert